deploy/roles/augur-build-exporter/grants.sql

```diff
@@ -0,0 +1,15 @@
+-- Deploy seattleflu/id3c-customizations:roles/augur-build-exporter/grants to pg
+
+begin;
+
```

Comment:
I got the role/grants reworkable pattern a little wrong when I introduced it in id3c. I fixed it for the roles I touched in the `locations` branch, but the gist is that the deploy change should be revoking everything and then regranting only specific stuff. This then works well when the change is reworked later and the deploy becomes revert as it doesn't require updating.

So for example, before the first grant in this file you'd write:

```sql
revoke all on database :"DBNAME" from "augur-build-exporter";
revoke all on schema receiving, warehouse, shipping from "augur-build-exporter";
revoke all on all tables in schema receiving, warehouse, shipping from "augur-build-exporter";
```